docs/tutorials/first-steps.rst

```diff
@@ -29,6 +29,7 @@ We'll start with a relatively simple data model:
 
 Using the `httpie <http://httpie.org>` tool we can post a sample record in the
 ``tasks`` collection:
+For proxy issues we can use  "http --proxy=http:http://usrname:passwrd@host:port COMMAND" instead of "http COMMAND"
```

Comment:
Please update your ``greeshmab:patch-1`` branch to make this little change so that we can merge it :+1: